modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -47,21 +52,21 @@ User-defined labels and {product-title} labels are available on the following GC
 User-defined tags are attached to resources created by the {product-title} Image Registry Operator and not on the resources created by any other Operators or the Kubernetes in-tree components.
 
 User-defined tags are available on the following GCP resources:
-* Storage bucket
+* Storage buckets
+* Compute instances
+* Compute disks
```

Comment:
This isn't rendering correctly in the preview:
<img width="713" alt="image" src="https://github.com/openshift/openshift-docs/assets/3442316/8370e621-f42b-4b01-a224-5727403cdcfe">
Might need a blank line before the list:
```suggestion

* Storage buckets
* Compute instances
* Compute disks
```

Reply:
ack.